rules/scripting-user-agents.data

```diff
@@ -46,3 +46,13 @@ winhttp.winhttprequest
 # https://kong.github.io/unirest-java/
 # User-Agent: unirest-java/VERSION
 unirest-java/
+# http library (deprecated since 2022)
+# https://index.scala-lang.org/scalaj/scalaj-http
+# User-Agent: scalaj-http/VERSION
+scalaj-http/
+# http library
+# https://http4s.org/
+# User-Agent: http4s-ember
+# http library
```

Comment:
Remove these three lines. The user agent for sttp is most likely the curl user agent (sttp uses curl as a backend).